internal/database/migrate.go

```diff
@@ -0,0 +1,115 @@
+package database
+
+import (
+	"context"
+	"strings"
+
+	"github.com/rs/zerolog/log"
+	"github.com/zibbp/ganymede/ent"
+	"github.com/zibbp/ganymede/internal/utils"
+)
+
+// VideosDirMigrate migrates the videos directory if it has changed.
+// It will do nothing if the videos directory has not changed.
+func (db *Database) VideosDirMigrate(ctx context.Context, videosDir string) error {
```

Comment:
This needs to handle the image path for channels as well

Reply:
Thanks, resolved this.